src/python/tests/metadata/test_prepare_genome.py

```diff
@@ -35,10 +35,13 @@
 
 
 class TestSchemas:
+    test_data_dir = Path()
+    tmp_dir = Path()
```

Comment:
```suggestion
    test_data_dir: Path
    tmp_dir: Path
```
Although not 100%, it seems based on the information I have been able to find that this should be ok :crossed_fingers: